src/bundles/comms.stanza

```diff
@@ -111,10 +111,23 @@ public pcb-bundle can-logical :
 ; SPI - Serial Peripheral Interface
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
+doc: \<DOC>
+SPI Bundle Pin Names
+A list of possible names for the SPI interface. Note that in order to connect
+SPI interfaces automatically, the constituent pins need to be present on all
+objects to be connected. 
+<DOC>
+
 public pcb-enum jsl/bundles/comms/SPIPins :
   SPI-CS
   SPI-MISO
   SPI-MOSI
+  SPI-COPI
+  SPI-CIPO
+  SPI-RDY
+  SPI-SDO
+  SPI-SDI
```

Comment:
Same comment - I think having multiple different ways to express the same bundle is an anti-pattern. Lets please remove these or create a different bundle type.

The reason is because of our "quasi-inheritance" bundle mechanism. If we have several different components with bundles that all have different names, that mechanism will never work correctly.